doc/DESIGN-AND-ROADMAP.md

```diff
@@ -61,6 +61,61 @@ Privacy Badger:
 
 #### Further Details
 
+Datastructures:
+
+action_map = { 'google.com': blocked, 'fonts.google.com', 'cookieblocked', 'apis.fonts.google.com': 'user_cookieblock', 'foo.tracker.net': 'allow', 'tracker.net': 'DNT', }
+snitch_map = {google.com: array('cooperq.com', 'noah.com', 'eff.org'), tracker.net: array(a.com, b.com, c.com)} 
+dnt_domains = array('tracker.net', 'dnt.eff.org')
+settings = {social_widgets = true, ...}
+cookie_block_list = "{'fonts.google.com': true, 'maps.google.com', true}"
+
+
+On Request():
+  check if privacy badger is enabled for this fqdn, if not then return
+  check if the fqdn is third party, if not then return
```

Comment:
does this mean PB won't block third parties on which PB is disabled?
So like if I disable pb on twitter and then there's an embedded twitter thing will it load or will it get blocked (assuming it's hueristic blocked)?